source/client-side-encryption/client-side-encryption.rst

```diff
@@ -633,22 +639,24 @@ The below steps should be taken:
 1. Let `U` be a new URL, initialized from the URL string
    :ts:`"http://169.254.169.254/metadata/identity/oauth2/token"`
 2. Add a query parameter ``api-version=2018-02-01`` to `U`.
-3. Add a query parameter ``resource=http://vault.azure.com/`` to `U`.
+3. Add a query parameter ``resource=https://vault.azure.net/`` to `U`.
 4. Prepare an HTTP GET request `Req` based on `U`.
 
    .. note:: All query parameters on `U` should be appropriately percent-encoded
 
 5. Add HTTP headers ``Metadata: true`` and ``Accept: application/json`` to
    `Req`.
 6. Issue `Req` to the Azure IMDS server ``168.254.169.254:80``. Let `Resp` be
-   the response from the server.
-7.  If `Resp_{status} ≠ 200`, obtaining the access token has failed, and the
-    HTTP response body of `Resp` encodes information about the error that
-    occurred. Return an error instead of an access token.
-8.  Otherwise, let `J` be the JSON document encoded in the HTTP response body
-    of `Resp`.
-9.  The result access token `T` is given as the ``access_token`` string property
-    of `J`. Return `T` as the resulting access token.
+   the response from the server. If the HTTP response is not completely received
+   within three seconds, consider the request to have timed out, and return an
+   error instead of an access token.
+7. If `Resp_{status} ≠ 200`, obtaining the access token has failed, and the HTTP
```

Comment:
can we formulate this like it was done for gcp (lines 607-608 above) https://github.com/mongodb/specifications/pull/1314/files#diff-de01d398114f63480af23798e09ffca599aa758efd04f15a9beec89b4566041bR606-R607

and in particular include this part: 
>return an error including the body of the HTTP response in the error message.